.templates/nodered/service.yml

```diff
@@ -1,6 +1,10 @@
 nodered:
   container_name: nodered
-  build: ./services/nodered/.
+  build:
+    context: ./services/nodered/.
+    args:
+    - DOCKERHUB_TAG=latest
+    - EXTRA_PACKAGES=
```

Comment:
Intentionally left blank?

Reply:
Yes. Does nothing when it is like that. But, if it bugs you, feel free to comment it out.